src/pages/AddItem.js

```diff
@@ -2,26 +2,33 @@ import React, { useState, useEffect } from 'react';
 import { withFirestore } from 'react-firestore';
 import AddItemHeader from './AddItemHeader';
 import getToken from '../lib/token';
-import NavTabs from '../components/NavTabs';
+
+const expectedPurchase = { soon: 7, kindOfSoon: 14, notSoon: 30 };
 
 const AddItem = ({ firestore }) => {
   const [name, setName] = useState('');
 
+  const [nextExpectedPurchase, setNextExpectedPurchase] = useState(0);
+
   const initialToken = () => window.localStorage.getItem('token') || getToken();
```

Comment:
I know these lines weren't part of the PR, but let's cleanup here since we're in the file. 

We're not setting the token in this component, so we don't need the `useEffect`. I might go with: 
```js
const [token] = useState(window.localStorage.getItem('token'))
```

Reply:
@stevelikesmusic I don't see how we can remove this because this is the only place where the token is set right now. I might not be understanding, but without that line how does the token get saved the first time in local storage? We don't have another component that handles it.

Reply:
You're totally right @prophen! My mistake. I was thinking that logic was in a different component